### ISSUE TITLE
Difficulty selecting precise written and audio sentences 

### ISSUE THREAD

**Issue by Shashanyan** [asbplayer/asbplayer#398]
Hi, 
I have been struggling to export the exact sentence I want and especially to extract the exact audio piece  I want from YTB videos. 
There, the subtitles are rarely divided into sentences (especially EU languages), thus one phrase starts and the end finishes in another. For the sentence, I've tried hold and drag from the side panel but I honestly don't really get where I'm supposed to drag it to, and using the toggle on the export panel isn't accurate to get the specific audio sentence without extra words. 

Is there a way around this please? or am I missing sth perhaps?
A Million thanks ! :) 

(if the export could be more precise it would be such an amazing tool because it's still working very well !!)

**Comment by killergerbah** [comment]
If you can link a YT video + timestamp I'll be able to precisely understand what you're reporting and can try to suggest workarounds.

Beyond that, precise audio recording is something that asbplayer can do a lot better and there are tasks in the TODO list to help with this such as displaying wave forms on the Anki dialog and intelligently detecting sentence boundaries across subtitles. However, these features may not come in the near short term.

**Comment by Shashanyan** [comment]
@killergerbah Thank you so much for the answer ! I think you understood well my issue, but I won't bother you any further with it as you are saying that it's a feature that will be improved in the future so for now I guess we just need to wait.

EXAMPLE : 
If you still want to have an example of what I meant :
On this YTB video : https://www.youtube.com/watch?v=ZRm7FDGz7kA&t=101s
I want to create an ANKI audio flashcard with the exact sentence "Contrairement à d’autres manuscrits, qui restent à l'Institut de France".
This sentence starts at 7:17min and finishes at 7:20. (divided in 2 different subtitles)

PROBLEM :
I struggle to extract exactly this part of the sentence as an audio to add it to Anki. The pb is that It starts either too early or finishes too late when I try to adjust with the red toggle (at the very bottom of export menu opened with ctrl+shift+x)

>I wish we could more precisely be able to choose when the audio recording starts and end basically. Anyways, wishing you best of luck with the improvement of the features :) The concept is really amazing.

**Comment by Shashanyan** [comment]
My workaround for now if it helps anyone :
-I copy paste the extact sentence I want from YTB transcript
-I find the exact beginning of the sentence in the video
-then I record the audio manually with ANKI's feature we can use when creating a card. I open ANKI card tab, click record, go to YTB, play the video, then pause the video when the desired sentence end and then go back to ANKI to stop the recording.

**Comment by killergerbah** [comment]
Got it I understand your problem now and thanks for reporting it. Yes it seems like something that could improved in asbplayer by merging together lines that appear to be part of the same sentence.

One suggestion I have based on the workaround you are using: maybe you didn't notice this but asbplayer has a manual audio recording bind as well.

**Comment by Shashanyan** [comment]
Right, I have tried using the shortcut to record the audio via asbplayer manually, but it doesn't always work, many times nothing happens. Thanks a lot for the suggestion though :)

**Comment by QuangKhanh34th** [comment]
got the same problem as yours and also tried the workaround, it works perfectly! I think this should be disclosed in the readme
